### ISSUE TITLE
Input field resets to its default value

### ISSUE THREAD

**Issue by lindhe** [home-assistant/frontend#12754]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

If I type text into an input field for a service, and the input field has a default value, I cannot clear the input field. 

### Describe the behavior you expected

When I delete the last character, the input field should become empty so I can type in a new string (not reset to the default value).

### Steps to reproduce the issue

There's almost certainly a shorter path to reproduce this issue, but I can't come up with one right now. Please share a better suggestion if you have one, this guide requires a Todoist account.

1. Install the Todoist Integration
2. Go to Developer Tools
3. Select Services
4. Use `todoist.new_task`
5. Click "Go to UI mode"
6. Check the "Project" box
7. Place the cursor at the end of "Inbox" and hit backspace.
8. Inbox --> Inbo --> Inb --> In --> I --> Inbox
9. Repeat step 8 until insanity occurs.

### What version of Home Assistant Core has the issue?

core-2022.5.5

### In which browser are you experiencing the issue with?

Firefox 100.0.2

### Which operating system are you using to run this browser?

Ubuntu 20.04.3 LTS (Focal Fossa)

### Additional information

Basically the same issue as #11047 except that it got resolved a while back.

**Comment by lindhe** [comment]
Still an issue.

**Comment by lindhe** [comment]
Still an issue.

**Comment by karwosts** [comment]
This should be fixed now. Now if you backspace the whole field, it will remain empty, and show a ghost value of "inbox" behind the cursor, but you can still type something else. 

If you find it not fixed, feel free to leave a comment and I will look again.

**Comment by lindhe** [comment]
Yep, works now. Fantastic. Thanks a lot! :)
